example/lib/presentation/search_grid_bloc/character_sliver_grid.dart

```diff
@@ -59,12 +57,33 @@ class _CharacterSliverGridState extends State<CharacterSliverGrid> {
             builderDelegate: PagedChildBuilderDelegate<CharacterSummary>(
               itemBuilder: (context, item, index) => CharacterGridItem(
                 character: item,
+                index: index,
               ),
             ),
+            initialBannerIndex: 6,
+            bannerFrequency: 3,
+            bannerWidgets: [
+              _buildBannerWidget(context, color: Colors.yellow),
+              _buildBannerWidget(context, color: Colors.blue),
+            ],
+            showBannerBetweenGridItems: true,
```

Comment:
As "between" grid items could mean any direction I would suggest to rename this to `showBannersBetweenGridRows`

Reply:
Hey @danielwilliamson - I addressed PR comment